src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -55,15 +56,18 @@ public void canCreateATitleLevelRequestForMultipleAvailableItemsAndAMatchingPick
     itemsFixture.basedUponDunkirkWithCustomHoldingAndLocation(holdings.getId(),
       null);
 
+    final String expectedPatronComment = "I need the book";
     JsonObject requestBody = createInstanceRequestObject(
       instanceMultipleCopies.getId(), requesterId, pickupServicePointId,
-      requestDate, requestExpirationDate);
+      requestDate, requestExpirationDate).put("patronComments", expectedPatronComment);
 
     Response postResponse = requestsFixture.attemptToPlaceForInstance(requestBody);
 
     JsonObject representation = postResponse.getJson();
     validateInstanceRequestResponse(representation, pickupServicePointId,
       instanceMultipleCopies.getId(), item2.getId(), RequestType.PAGE);
+
+    assertThat(representation, hasJsonPath("patronComments", expectedPatronComment));
```

Comment:
Why is this outside the `validateInstanceRequestResponse` method?

Reply:
Because the method is shared between all the tests and I'll have to refactor all them to populate the patron comment.

Reply:
What does `hasJsonPath` do with a null expected value?

Reply:
if the key is present in the json but has null value then it matches the assertion, but if the key does not exist then it throws an exception saying that path is not found.

Reply:
Ah, ok, then we would need a branch in the method. Let's leave it how it is.